usersession/userd/privileged_desktop_launcher.go

```diff
@@ -290,12 +315,14 @@ func parseExecCommand(command string, icon string) ([]string, error) {
 			arg = arg[1:]
 		} else if strings.HasPrefix(arg, "%") {
 			switch arg {
-			case "%f", "%F", "%u", "%U":
-				// If we were launching a file with
-				// the application, these variables
-				// would expand to file names or URIs.
-				// As we're not, they are simply
-				// removed from the argument list.
+			case "%f", "%u":
+				if (arguments != nil) && (len(arguments) > 0) {
```

Comment:
You can remove the `arguments != nil` check here, since it will behave like a zero length slice.

I think we should error out if there is more than one URI too, rather than silently losing the extra URIs.

Reply:
I wasn't suggesting to remove the if statement entirely here. Just that if `arguments == nil`, then `len(arguments) == 0`, so the `len(arguments) > 0` would evaluate to false.